### PR TITLE
Improve mnemonic instruction

### PR DESCRIPTION
**Pull request by lukaw3d**

### CODE REVIEW

File: src/locales/en/translation.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
     },
     "mnemonic": {
       "header": "Enter your keyphrase",
-      "instruction": "Enter your 12, 18 or 24 words keyphrase below, each words separated by spaces.",
+      "instruction": "Enter all your keyphrase words below, separated by spaces. Most keyphrases are made of either 24 words, or 12 words.",
```
Comment (matevz) [inline src/locales/en/translation.json:29]: Remove commas here too. I'll poke Raphael to update the French terms.

**Review by matevz — APPROVED** [review]
Some minor nits regarding commas. Otherwise, I like this change!

File: src/locales/en/translation.json (hunk 1 above)
Comment (matevz) [inline src/locales/en/translation.json:29]: Also please update French translation (kudos to Raphael):
```
Entrez ci-dessous tous les mots qui composent votre phrase secrète (séparés par des espaces). La plupart des phrases secrètes sont composées de 24 ou 12 mots.
```

Comment (matevz) [inline src/locales/en/translation.json:29]: Also please update French translation (kudos to Raphael):
```
Entrez ci-dessous tous les mots qui composent votre phrase secrète (séparés par des espaces). La plupart des phrases secrètes sont composées de 24 ou 12 mots.
```

Comment (lukaw3d) [reply src/locales/en/translation.json:29]: Yay

**Pull request merged**